freqtrade/maintenance_margin.py

```diff
@@ -0,0 +1,52 @@
+from typing import List
```

Comment:
I think maintenance margin is only relevant for leverage - so please move this to `freqtrade.leverage` (basically move it into the subdirectory).

Reply:
it creates a circular import, I think I might remove that sub directory also, and can hopefully move the formulas onto the exchange subclasses as class methods